src/nitro-contracts/bridge/Inbox.sol

```diff
@@ -0,0 +1,323 @@
+        }
+        return _deliverMessage(
+            L2_MSG,
+            // undoing sender alias here to cancel out the aliasing
+            AddressAliasHelper.undoL1ToL2Alias(msg.sender),
+            abi.encodePacked(
+                L2MessageType_unsignedEOATx,
+                gasLimit,
+                maxFeePerGas,
+                nonce,
+                uint256(uint160(address(100))), // ArbSys address
+                value,
+                abi.encodeWithSelector(ArbSys.withdrawEth.selector, withdrawTo)
+            ),
+            0
+        );
+    }
+
+    /// @inheritdoc IInbox
+    function depositEth() public payable whenNotPaused onlyAllowed returns (uint256) {
+        address dest = msg.sender;
+
+        // solhint-disable-next-line avoid-tx-origin
+        if (AddressUpgradeable.isContract(msg.sender) || tx.origin != msg.sender) {
+            // isContract check fails if this function is called during a contract's constructor.
+            dest = AddressAliasHelper.applyL1ToL2Alias(msg.sender);
+        }
+
+        return _deliverMessage(L1MessageType_ethDeposit, msg.sender, abi.encodePacked(dest, msg.value), msg.value);
+    }
+
+    /// @notice deprecated in favour of depositEth with no parameters
+    function depositEth(uint256) external payable whenNotPaused onlyAllowed returns (uint256) {
+        return depositEth();
+    }
+
+    /**
+     * @notice deprecated in favour of unsafeCreateRetryableTicket
+     * @dev deprecated in favour of unsafeCreateRetryableTicket
+     * @dev Gas limit and maxFeePerGas should not be set to 1 as that is used to trigger the RetryableData error
+     * @param to destination L2 contract address
+     * @param l2CallValue call value for retryable L2 message
+     * @param maxSubmissionCost Max gas deducted from user's L2 balance to cover base submission fee
+     * @param excessFeeRefundAddress gasLimit x maxFeePerGas - execution cost gets credited here on L2 balance
+     * @param callValueRefundAddress l2Callvalue gets credited here on L2 if retryable txn times out or gets cancelled
+     * @param gasLimit Max gas deducted from user's L2 balance to cover L2 execution. Should not be set to 1 (magic value used to trigger the RetryableData error)
+     * @param maxFeePerGas price bid for L2 execution. Should not be set to 1 (magic value used to trigger the RetryableData error)
+     * @param data ABI encoded data of L2 message
+     * @return unique message number of the retryable transaction
+     */
+    function createRetryableTicketNoRefundAliasRewrite(
+        address to,
+        uint256 l2CallValue,
+        uint256 maxSubmissionCost,
+        address excessFeeRefundAddress,
+        address callValueRefundAddress,
+        uint256 gasLimit,
+        uint256 maxFeePerGas,
+        bytes calldata data
+    )
+        external
+        payable
+        whenNotPaused
+        onlyAllowed
+        whenRefundAddressAllowed(to, excessFeeRefundAddress, callValueRefundAddress)
```

Comment:
I think maybe this modifier only needs to be added to unsafeCreateRetryable ticket and thats it

Reply:
This one is not needed I believe